src/pymovements/stimulus/image.py

```diff
@@ -22,9 +22,9 @@
 
 from pathlib import Path
 
+from pymovements._utils._strings import curly_to_regex
```

Comment:
do we need the double underscore for both `_utils` and `_strings`? from my pov `utils` already implies no user interaction